e2e-tests/commands/insurance/complete-and-submit-alternative-currency-form.js

```diff
@@ -5,11 +5,12 @@ const { CURRENCY_CODE } = INSURANCE_FIELD_IDS.CURRENCY;
 
 /**
  * completeAndSubmitAlternativeCurrencyForm
- * presses hyperlink and completes the alternative currency form
+ * conditionally clicks "alternative currency" hyperlink and completes the alternative currency form
  * if isoCode provided - clicks the radio for currency and submits alternative currency form
  * if alternativeCurrency true, then enters alternative currency and submits alternative currency form
- * @param {String} isoCode: isoCode provided for radio selection
- * @param {Boolean} alternativeCurrency: if alternative currency should be entered
+ * @param {String} isoCode: isoCode for radio selection.
+ * @param {Boolean} alternativeCurrency: If alternative currency should be entered.
+ * @param {Boolean} clickAlternativeCurrencyLink: if the "Alternative currency" link should be clicked.
```

Comment:
can this be removed yet?

Reply:
Nope - need to tackle ~5+ new tickets first. There is a separate ticket to remove this.